packages/core/admin/admin/src/pages/MarketplacePage/tests/plugins.test.js

```diff
@@ -94,12 +106,8 @@ describe('Plugins tab', () => {
   });
 
   it('should return plugin search results matching the query', () => {
-    const { container } = render(App);
-    const pluginsTab = screen.getByRole('tab', { name: /plugins/i });
-    fireEvent.click(pluginsTab);
-
-    const input = getByPlaceholderText(container, 'Search');
-    fireEvent.change(input, { target: { value: 'comment' } });
+    const input = getByPlaceholderText(renderedContainer, 'Search');
```

Comment:
Since there's no waitForReload() I guess this only works thanks to react-query caching the request?

If that's the case I don't find it ideal as it means this test depends on the one above to run

Reply:
Damn yeah, I guess actually all the tests need the cache to reset to be truly isolated so I'll probably remove the nested describe, clear the cache and waitForReload beforeEach